text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+
+### Intro 
+When considering what questions to ask, we considered a few things: 
+1. What questions do we have to ask ourselves every time we create a new Ember app?
+1. How many questions do we think developers will tolerate?
+1. Should the questions be different if it is an app vs addon?
+
+We will also endeavor to provide strong defaults for these questions, based on the results from the community surveys we have been conducting over the past few years and the defaults as stated by Ember itself (either explicitly or implicitly). **If these defaults change over time as the result of community RFCs, the default values or other responses for the interactive questions should be updated as part of those RFCs.**
+
+So, how then do we decide which questions to include and which not to include? It became necessary to then develop a rubric by which the proposed list of questions could be measured. 
+
+### Question Rubric
+We intend for this section to be normative; that is, we intend for it to define how questions are evaluated for inclusion both now and in the future. 
+
+Evaluating criteria: 
+1. Is it hard (or time-consuming) to change it later? *For example, it's easy to remove the `ember-welcome-page` addon, so we did not include it. But it's time consuming to change from `yarn` to `npm` (or vice-versa) so we made sure to include that.* 
+1. Is it difficult to discover the option?
+1. What is the maintenance cost? *For example, a nice question to add could be something like, Do you want to use a CSS preprocessor? But, because Ember does not deal with SASS, LESS... by default, the interactive workflow will have to install an addon (like `ember-cli-less`) which would imply that the CLI team will have to ensure it never fails.* 
+1. Does the question affect another one? *For example, the LTS question can affect the CI file generated by the CI question (setup tests on CI with `ember-try`). Questions that modify the result of other questions too much should be avoided to prevent high maintenance.*
+
+### Triggering the Workflow
+
+The following commands, and reasonable combinations thereof, should work to enter into the interactive workflow: 
+
+1. `ember new`
+1. `ember new --interactive`
+1. `ember new -i`
+1. `ember new my-app -i`
+
+### App/Addon 
+
+Starting a new Ember app or addon as we do now (at the time of this writing) will not change. If a user types `ember new my-app` they will still get a new Ember app generated in the usual ways. However, if the user types `ember new` they will enter into an interactive workflow that will ask them a few key questions about their application. 
+
+If a user were to type in existing flags that are also questions in the interactive workflow (i.e., `ember new my-app --lang en-US --interactive`), it will skip the question entirely. 
```

Comment:
I don't think this'd be a technical limitation. I can think of backing the interactive wizard with a statechart that would determine which questions to ask based on the presence or lack of flags passed at invocation time